tests/TestHarness/Cluster.py

```diff
@@ -486,32 +462,37 @@ def connectGroup(group, producerNodes, bridgeNodes) :
                     break
         Cluster.__LauncherCmdArr = cmdArr.copy()
 
-        s=" ".join([("'{0}'".format(element) if (' ' in element) else element) for element in cmdArr.copy()])
-        if Utils.Debug: Utils.Print("cmd: %s" % (s))
-        if 0 != subprocess.call(cmdArr):
-            Utils.Print("ERROR: Launcher failed to launch. failed cmd: %s" % (s))
-            return False
+        launcher = cluster_generator(cmdArr[2:])
+        launcher.define_network()
+        launcher.generate()
+        self.nodes = []
+        for instance in launcher.network.nodes.values():
+            eosdcmd = launcher.construct_command_line(instance)
+
+            nodeNum = instance.index
+            node = Node(self.host, self.port + nodeNum, nodeNum, Path(instance.data_dir_name), Path(instance.config_dir_name), eosdcmd, unstarted=instance.dont_start, launch_time=launcher.launch_time, walletMgr=self.walletMgr, nodeosVers=self.nodeosVers)
```

Comment:
If you broke the line above into 2, please do the same here.

Reply:
Done.